### PR TITLE
i18n(es): Updated `components.mdx`, `customization.mdx`, `configuration.mdx`, `frontmatter.mdx` & `index.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `components.mdx`, `customization.mdx`, `configuration.mdx` & `frontmatter.mdx`

`components.mdx`
* [`dc42569`](https://github.com/withastro/starlight/commit/dc42569bddfae2c48ea60c0dd5cc70643a129a68#diff-bde0eef3b5aa6eb8762ac30b4992c55505bc9eb0373f4a05d81f58f39c05e3bd)

`customization.mdx`
* [`5db3e6e`](https://github.com/withastro/starlight/commit/5db3e6ea2e5cb7d9552fc54567811358851fb533#diff-596fab03bd0326b4a66f8afc7a97400b00fc458bdf93e1a32bcd72be7b8790f2)
* [`2598b8f`](https://github.com/withastro/starlight/commit/2598b8ffe2754f905c0e10b2479b0065cc70494c)

`configuration.md`
* [`69b7d4c`](https://github.com/withastro/starlight/commit/69b7d4c23761a45dc2b9ea75c6c9c904a885ba5d#diff-258b8e475fefcd9503466ed259ff2913e1dd52a90fd46170aca60318c67459ce)
* [`5db3e6e`](https://github.com/withastro/starlight/commit/5db3e6ea2e5cb7d9552fc54567811358851fb533#diff-258b8e475fefcd9503466ed259ff2913e1dd52a90fd46170aca60318c67459ce)

`frontmatter.md`
* [`69b7d4c`](https://github.com/withastro/starlight/commit/69b7d4c23761a45dc2b9ea75c6c9c904a885ba5d#diff-6d478912950527786985d16b5fef998fc9a98ffc282f20ebe557b822d0e717bd)

### CODE REVIEW

File: docs/src/content/docs/es/guides/components.mdx [hunk 1]
```diff
@@ -29,6 +29,20 @@ import OtroComponente from '../../componentes/OtroComponente.astro';
 
 Debido a que Starlight está impulsado por Astro, puedes agregar soporte para componentes construidos con cualquier [framework UI compatible (React, Preact, Svelte, Vue, Solid, Lit y Alpine)](https://docs.astro.build/en/core-concepts/framework-components/) en tus archivos MDX.
 
+### "Compatibilidad con los estilos de Starlight"
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:32]: I guess we shouldn’t have the `"` here?

```suggestion
### Compatibilidad con los estilos de Starlight
```

File: docs/src/content/docs/es/guides/components.mdx [hunk 2]
```diff
@@ -29,6 +29,20 @@ import OtroComponente from '../../componentes/OtroComponente.astro';
...
+
+Starlight aplica estilos predeterminados a tu contenido de Markdown, por ejemplo, agregando márgenes entre elementos.
+Si estos estilos entran en conflicto con la apariencia de tu componente, establece la clase `not-content` en tu componente para desactivarlos.
+
+```astro
+---
+// src/components/Example.astro
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:39]: We can probably translate this:

```suggestion
// src/components/Ejemplo.astro
```

**Review by delucis — COMMENTED** [review]
Thanks for the speedy update @Waxer59! Just spotted two details:

**Comment by Waxer59** [comment]
Done! thanks for the review 🤠

**Review by delucis — APPROVED** [review]
Looks great. Spanish docs: **_COMPLETE_** 💥

**Pull request merged**